domain/model/service/modelservice.go

```diff
@@ -20,16 +21,23 @@ type ModelState interface {
 	Delete(ctx context.Context, uuid coremodel.UUID) error
 }
 
+// ModelsState represents the state required for reading all model information.
+type ModelsState interface {
+	Get(context.Context, coremodel.UUID) (coremodel.Model, error)
+}
+
 // ModelService defines a service for interacting with the underlying model
 // state, as opposed to the controller state.
 type ModelService struct {
-	st ModelState
+	modelsST ModelsState
+	st       ModelState
```

Comment:
Oh I see. Can we call it `ModelGetter` then. `ModelsState` is bleh IMO.

Reply:
Done & Done